test/integration/policy_info_metric_test.go

```diff
@@ -71,37 +78,63 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test policy_governance_info metric
 		routeHost := routeList.Items[0].Object["spec"].(map[string]interface{})["host"].(string)
 		By("Got the metrics route url: " + routeHost)
 		propagatorMetricsURL = "https://" + routeHost + "/metrics"
+	})
+	It("Sets up a ServiceAccount without permissions for metrics", func() {
+		_, err := common.OcHub("create", "serviceaccount", noMetricsAccName, "-n", userNamespace)
+		Expect(err).To(BeNil())
 
-		// get auth token from service account
-		By("Setting up ServiceAccount for authentication")
-		_, err = common.OcHub("create", "serviceaccount", saName, "-n", userNamespace)
+		_, err = common.OcHub("apply", "-f", noMetricsTokenYaml, "-n", userNamespace)
 		Expect(err).To(BeNil())
-		_, err = common.OcHub("create", "clusterrolebinding", roleBindingName, "--clusterrole=cluster-admin", fmt.Sprintf("--serviceaccount=%s:%s", userNamespace, saName))
+
+		var encodedtoken string
+
+		// The secret could take a moment to be populated with the token
+		Eventually(func(g Gomega) {
+			var err error
+			encodedtoken, err = common.OcHub("get", "secret", noMetricsTokenName,
```

Comment:
Optional: It'd be cleaner to use the Go client and then you wouldn't have to check the length of the string to determine if the token is present or not.